modules/manual-configuration-image-registry-ocp.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="manual-configuration-image-registry-ocp_{context}"]
+= Manually configuring {ocp} registry
+
+You can integrate {product-title} with {ocp} built-in container image registry.
+
+.Prerequisites
+* You need a *Username* and a *Password* for authentication with the {ocp} registry.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Image Integrations* section, select *Generic Docker Registry*.
+The *Configure image integration* modal box opens.
+. Click *New Integration*.
+. Enter the required details for:
+.. *Integration Name*: The name you want to give to this integration.
+.. *Types*: Select *Registry*.
+.. *Endpoint*: The address of the registry.
```

Comment:
More nits.

For the *Integration Name* field, why not "The name of the integration."?  That is the spare style you have for the *Endpoint* field.  It's really that my eye caught "want to give."